packages/rancher-istio/charts/templates/istio-install-job.yaml

```diff
@@ -32,6 +40,9 @@ spec:
             mountPath: /app/overlay-config.yaml
             subPath: overlay-config.yaml
           {{- end }}
+          securityContext:
+            runAsUser: 101
+            runAsGroup: 101 
```

Comment:
Should this be configurable (and also update the PSP)? Why was it set to 101?

Reply:
nit: why not just leave this where it originally was under serviceAccountName?

Reply:
Moved back to global job setting instead of per contian. This should not be configurable because the istio-installer image does not run as root so I do not want people to attempt to run the container as root. The user in the istio-installer is a 101 user.

Reply:
https://github.com/nginxinc/docker-nginx/blob/d496baf859613adfe391ca8e7615cc7ec7966621/mainline/alpine/Dockerfile#L17 additional context of how the user is generated - this is from the alpine image, but its the same across all nginx user setups.

Reply:
ah yes this makes sense. Thanks for the clarification!